website/docs/guides/cloud_integrations_guide.html.markdown

```diff
@@ -28,7 +28,7 @@ Add the following module to your Terraform code, and set the variables to your d
 
 ```
 module "newrelic-aws-cloud-integrations" {
-  source = "github.com/newrelic/terraform-provider-newrelic/tree/main/examples/modules/cloud-integrations/aws"
+  source = "github.com/newrelic/terraform-provider-newrelic/examples/modules/cloud-integrations/aws"
```

Comment:
A note for @kidk: Discarding `/tree/main` from these code snippets as I've been able to identify the link to import modules works without `/tree/main`. This was also identified via [this GitHub issue](https://github.com/newrelic/terraform-provider-newrelic/issues/2393).